Ponca/src/Fitting/mean.h

```diff
@@ -73,6 +73,15 @@ namespace Ponca {
         PONCA_EXPLICIT_CAST_OPERATORS(MeanNormal,meanNormal)
         PONCA_FITTING_DECLARE_INIT
         PONCA_FITTING_DECLARE_ADDNEIGHBOR
+
+        /// \brief Mean of the normals of the input points
+        ///
+        /// Defined as \f$ n(\mathbf{x}) = \frac{\sum_i w_\mathbf{x}(\mathbf{p_i}) \mathbf{n_i}}{\sum_i w_\mathbf{x}(\mathbf{p_i})} \f$,
+        ///  where \f$\left[\mathbf{p_i}, \mathbf{n_i} \in \text{neighborhood}(\mathbf{x})\right]\f$ are all the point and normal samples in \f$\mathbf{x}\f$'s neighborhood
+        PONCA_MULTIARCH inline VectorType mean() const {
```

Comment:
"meanNormal" creates compilation errors, because the name of the class is the same. Maybe is it better to call the function as a getter like : "getMeanNormal" ?

Reply:
The `get` prefix is not really used in Ponca, but I don't see a better solution, (apart from renaming the `PONCA_EXPLICIT_CAST_OPERATORS(...,asMeanNormal)`  which would also introduce a change in the naming convention)

Reply:
I see, 
Another solution would be to compute the mean Normal inside the `meanNormalDer`, when computing the `dMeanNormal()`.
So the function `getMeanNormal()` will not be existing anymore.

But this also breaks the behavior previously done with the `meanPosition` and  `meanPositionDer` classes.